packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -133,6 +135,35 @@ const TradingAppCard = ({
     const is_disabled = !!(mt5_acc_auth_status && !migration_status) && !is_eu_user;
     const platform_name = is_account_being_created ? name : sub_title ?? name;
 
+    const renderStatusBadge = () => {
```

Comment:
@shontzu please move out this to a different component.

Reply:
We don't need to have a separate function, all we need is a new `MT5_ACCOUNT_STATUS.disabled`